### ISSUE TITLE
No normalization in demo/visualize_result.py

### ISSUE THREAD

**Issue by jsheo96** [JDAI-CV/fast-reid#733]
In demo/visualie_result.py file there is following code snippet at line 125~126

```
 # compute cosine distance
    distmat = 1 - torch.mm(q_feat, g_feat.t())
```

I think there should be a nomalization before torch.mm like "postprocess" function in demo/demo.py

The resulting jpg outputs also show the similarities over 1, which is weird behavior.

**Comment by jayong-sv** [comment]
@jsheo96  I have the same question. How did you solve it?

Here is another issue about this problem: https://github.com/JDAI-CV/fast-reid/issues/651

**Comment by jayong-sv** [comment]
I added `feats = F.normalize(feats)` after [here](https://github.com/JDAI-CV/fast-reid/blob/c9bc3ceb2f7a6438b62fb515ea3df6d1e999e95d/demo/visualize_result.py#L117), and it seems like working fine.
